docs/src/data/globalnav.ts

```diff
@@ -32,6 +32,12 @@ export const LEFT_NAV_LINKS = [
     url: 'https://ui.docs.amplify.aws/',
     order: 3,
   },
+  {
+    type: 'DEFAULT',
+    label: 'Contribute',
+    url: 'https://docs.amplify.aws/contribute',
```

Comment:
nit: looking across other url, they all have a trailing `/`

Reply:
@katiegoines Please add the trailing URL, since docs site returns a 301 for urls with missing trailing `/`, which will break out broken link checker.